cdc/sink/dmlsink/txn/mysql/mysql.go

```diff
@@ -68,6 +68,8 @@ type mysqlBackend struct {
 	metricTxnSinkDMLBatchCallback prometheus.Observer
 	// implement stmtCache to improve performance, especially when the downstream is TiDB
 	stmtCache *lru.Cache
+	// Indicate if the CachePrepStmts should be enabled or not
+	cachePrepStmts bool
```

Comment:
Can we just access it by `cfg.CachePrepStmts`? Because we already have the cfg in this struct.

Reply:
when on serverside, max_prepared_stmt_count is 0 or int(maxPreparedStmtCount/(cfg.WorkerCount+1)) is 0, even cfg.CachePrepStmts is true, we still need to disable prep stmt cache. So a new field cachePrepStmts is added to control if prep stmt cache should be used or not.

Reply:
I'm not sure if the value of cfg.CachePrepStmts will get reset or not, when the downstream MySQL config is changed and CDC need to rebuild the connection pool, or the sink URL options and the worker count are updated. So for safe a new field cachePrepStmts is added.